PWGHF/TableProducer/candidateCreatorOmegac.cxx

```diff
@@ -0,0 +1,525 @@
+  OutputObj<TH1F> hInvMassOmegac{TH1F("hInvMassOmegac", "Omegac invariant mass;inv mass;entries", 500, 2.2, 3.1)};
+  OutputObj<TH1F> hMassOmegacNotFixed{TH1F("hMassOmegacNotFixed", "hMassOmegacNotFixed;invmass;entries", 500, 2.2, 3.1)};
+
+  void init(InitContext const&)
+  {
+    ccdb->setURL(ccdbUrl);
+    ccdb->setCaching(true);
+    ccdb->setLocalObjectValidityChecking();
+    lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbPathLut));
+    if (!o2::base::GeometryManager::isGeometryLoaded()) {
+      ccdb->get<TGeoManager>(ccdbPathGeo);
+    }
+    runNumber = 0;
+  }
+
+  void process(SelectedCollisions::iterator const& collision,
+               aod::BCsWithTimestamps const& bcWithTimeStamps,
+               aod::CascDataExt const& cascades,
+               MyTracks const& tracks,
+               aod::V0Datas const&,
+               aod::V0sLinked const&)
+  {
+
+    // set the magnetic field from CCDB
+    auto bc = collision.bc_as<o2::aod::BCsWithTimestamps>();
+    initCCDB(bc, runNumber, ccdb, isRun2 ? ccdbPathGrp : ccdbPathGrpMag, lut, isRun2);
+    auto magneticField = o2::base::Propagator::Instance()->getNominalBz(); // z component
+
+    // 2-prong vertex fitter to build the omegac vertex
+    o2::vertexing::DCAFitterN<2> df;
+    df.setBz(magneticField);
+    df.setPropagateToPCA(propagateToPCA);
+    df.setMaxR(maxR);
+    df.setMaxDZIni(maxDZIni);
+    df.setMinParamChange(minParamChange);
+    df.setMinRelChi2Change(minRelChi2Change);
+    df.setUseAbsDCA(useAbsDCA);
+    df.setWeightedFinalPCA(useWeightedPCA);
+    df.setRefitWithMatCorr(refitWithMatCorr);
+
+    // 2-prong vertex fitter to build the cascade vertex
+    o2::vertexing::DCAFitterN<2> dfc;
+    dfc.setBz(magneticField);
+    dfc.setPropagateToPCA(propagateToPCA);
+    dfc.setMaxR(maxR);
+    dfc.setMaxDZIni(maxDZIni);
+    dfc.setMinParamChange(minParamChange);
+    dfc.setMinRelChi2Change(minRelChi2Change);
+    dfc.setUseAbsDCA(useAbsDCA);
+    dfc.setWeightedFinalPCA(useWeightedPCA);
+    dfc.setRefitWithMatCorr(refitWithMatCorr);
+
+    // 2-prong vertex fitter to build the V0 vertex
+    o2::vertexing::DCAFitterN<2> dfv;
+    dfv.setBz(magneticField);
+    dfv.setPropagateToPCA(propagateToPCA);
+    dfv.setMaxR(maxR);
+    dfv.setMaxDZIni(maxDZIni);
+    dfv.setMinParamChange(minParamChange);
+    dfv.setMinRelChi2Change(minRelChi2Change);
+    dfv.setUseAbsDCA(useAbsDCA);
+    dfv.setWeightedFinalPCA(useWeightedPCA);
+    dfv.setRefitWithMatCorr(refitWithMatCorr);
+
+    double massPionTrueValue = RecoDecay::getMassPDG(kPiPlus);    // pdg code 211
+    double massProtonTrueValue = RecoDecay::getMassPDG(kProton);  // pdg code 2212
+    double massLambdaTrueValue = RecoDecay::getMassPDG(kLambda0); // pdg code 3122
+    double massXiTrueValue = RecoDecay::getMassPDG(kXiMinus);     // pdg code 3312
+    double massOmegacTrueValue = RecoDecay::getMassPDG(kOmegac0); // pdg code 4332
```

Comment:
Please remove the `TrueValue` suffixes. They serve no purpose. There is no such a thing as true or false values of a particle mass.